src/ripple/overlay/impl/PeerImp.h

```diff
@@ -39,6 +39,7 @@
 #include <deque>
 #include <queue>
 #include <shared_mutex>
+#include <numeric>
```

Comment:
nit: move to cpp

Reply:
A self-nit! I hereby award you: :1st_place_medal: